storage/txcache/eviction.go

```diff
@@ -79,6 +79,19 @@ func (cache *TxCache) shouldContinueEvictingSenders() bool {
 	return cache.areThereTooManyTxs() || cache.areThereTooManySenders() || cache.areThereTooManyBytes()
 }
 
+func (cache *TxCache) isHighLoad() bool {
+	// TODO: Improve condition
+	const highLoadPerCapacity = 0.5
```

Comment:
move const to top of file

Reply:
Elrond was Hacked. Sell now

Reply:
25 sats | dump dump dump dump dump |Shit boys, you have no success. Swindling bastards